src/typography/attributes.js

```diff
@@ -212,14 +212,42 @@ p5.prototype.textStyle = function(theStyle) {
  * describe('Letter P and p5.js are displayed with vertical lines at end.');
  * </code>
  * </div>
+ *
+ *
+ * <div>
+ * <code>
+ * textSize(28);
+ *
+ * let aChar = 'text\nWidth()';
+ * let cWidth = textWidth(aChar);
+ * text(aChar, 0, 40);
+ * line(cWidth, 0, cWidth, 50);
```

Comment:
I think changing the line length from 50px to 100px would still improve the documentation @munusshih!